sensor_msgs/Image image
---
geometry_msgs/Pose estimated_pose